ci/tests/test_wasm_warm_path_perf.py

```
@@ -0,0 +1,113 @@
+        self.assertEqual(h1, h2)
+
+    def test_second_call_is_faster_than_first(self) -> None:
+        t0 = time.perf_counter()
+        _compute_src_file_list_hash()
+        cold = time.perf_counter() - t0
+
+        t0 = time.perf_counter()
+        _compute_src_file_list_hash()
+        warm = time.perf_counter() - t0
+
+        # Memoized call must be at least 5× faster than the cold walk.
```

Ruff (0.15.13)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)